src/openrct2/world/map.c

```diff
@@ -2837,13 +2835,12 @@ void game_command_place_large_scenery(sint32* eax, sint32* ebx, sint32* ecx, sin
                 (tile_num << 10) |
                 entry_index;
 
-            new_tile_element->properties.scenerymultiple.colour[0] = colour1;
-            new_tile_element->properties.scenerymultiple.colour[1] = colour2;
+            scenery_large_set_primary_colour(new_tile_element, colour1);
```

Comment:
Are you sure the calling functions are not going to be putting any other information in colour1 and colour2?

Reply:
The only other kind of information stored there is the banner Id. And that is handled directly below.